### PR TITLE
Update free-courses-ar.md

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
Simple changes

### CODE REVIEW

File: courses/free-courses-ar.md [hunk 1]
```diff
@@ -19,6 +18,7 @@
 * [Game Development](#game-development)
 * [Git](#git)
 * [HTML and CSS](#html-and-css)
+    * [Bootstrap](#bootstrap)
```
Comment (davorpa) [inline courses/free-courses-ar.md:21]: Why this change in TOC but without move the resources in listing?

We need mantain the structure consistently

Comment (GH05T-HUNTER5) [reply courses/free-courses-ar.md:21]: Ok 
Sry
